src/lib/raft/raft.c

```diff
@@ -1165,7 +1165,8 @@ raft_stop_candidate(struct raft *raft)
 		 * Leader is seen and node is waiting for its death. Do not stop
```

Comment:
IMO this comment needs to be updated and include brief description why is_write_in_progress is also checked.

Reply:
Sure:
```diff
diff --git a/src/lib/raft/raft.c b/src/lib/raft/raft.c
index 11369e844..4dccc7aea 100644
--- a/src/lib/raft/raft.c
+++ b/src/lib/raft/raft.c
@@ -1163,7 +1163,8 @@ raft_stop_candidate(struct raft *raft)
        } else {
                /*
                 * Leader is seen and node is waiting for its death. Do not stop
-                * the timer.
+                * the timer. If there is a write in progress the timer is
+                * stopped now, but will be re-started once the write completes.
                 */
                assert(raft_ev_timer_is_active(&raft->timer) ||
                       raft->is_write_in_progress);   
```